js/ui/screenShield.js

```diff
@@ -459,7 +459,6 @@ var ScreenShield = class {
         }
 
         this._dialog.allowCancel = allowCancel;
-        this._dialog.grab_key_focus();
```

Comment:
Doesn't this interfere with the regular unlock dialog? Specifically, typing immediately after turning the screen on.

Reply:
Yeah I thought it might to but in practice it doesn't. I think that's because Main.pushModal() causes key focus to be grabbed for whatever is becoming modal.

Reply:
But despite what I said in the commit message, maybe it is best to keep this patch downstream since it doesn't provide any benefit upstream.